src/chia_log/handlers/daily_stats/stats_manager.py

```diff
@@ -101,3 +103,20 @@ def _run_loop(self):
 
     def stop(self):
         self._is_running = False
+
+    def _parse_notify_time(self, value, hm):
+        if type(value) == int:
+            if hm == "hour":
+                return value
+            return 0
+        elif type(value) == str:
+            if re.match('(?:[01]\d|2[0123]):(?:[012345]\d)', value) is not None:
+                if hm == "hour":
+                    return int(value[:2])
+                elif hm == "minute":
+                    return int(value[-2:])
+            else:
+                if hm == "hour":
+                    return 21
+                elif hm == "minute":
+                    return 0
```

Comment:
This method is a bit difficult to read and maintain for its functionality. (too many if-statements)
I'd propose to simplify it by returning a dict or tuple with hour & minutes instead.

It's also better design to explicitly specify the default value as an optional argument so it's not hidden away in the implementation and can be more easily changed. Please also add type annotations for function arguments.

I propose roughly something like:

```
from typing import Union

...

def _parse_notify_time(self, value: Union[str, int], default: dict = {'hour': 21, 'minute': 0}) -> dict:
    if type(value) == int:
        return {'hour': value, 'minute': 0}

    match = re.match('(?:[01]\d|2[0-3]):(?:[0-5]\d)', value)
    if match:
        return {'hour': int(value[:2]), 'minute': int(value[-2:])}

    return default
```

Reply:
@martomi Sorry about that, my first time into python. I've refactored it and thanks for the feedback!